doc/devdocs/core/settings/viewmodels.md

```diff
@@ -1,25 +1,25 @@
-# Viewmodels
-The viewmodels are located within the [`Settings.UI.Library`](/src/settings-ui/Settings.UI.Library) project.
+# View Models
+The view models are located within the [`Settings.UI.Library`](/src/settings-ui/Settings.UI.Library) project.
```

Comment:
The code items are ViewModel(s), so spelling it out the item with spaces seems reasonable.